forge-game/src/main/java/forge/game/Game.java

```diff
@@ -265,6 +265,7 @@ public Game(Iterable<RegisteredPlayer> players0, GameRules rules0, Match match0,
         for (RegisteredPlayer psc : players0) {
             IGameEntitiesFactory factory = (IGameEntitiesFactory)psc.getPlayer();
             Player pl = factory.createIngamePlayer(this, plId++);
+            pl.setHasLivingEnd(psc.getDeck().getMain().countByName("Living End", true) > 0);
```

Comment:
Would love to see this defined in the Deck file so it could potentially be reused for different deck strategies. But maybe we do that in the next deck.

Reply:
I think we cant do that on LDA generated decks since they are not saved, gonna add the Living End check on AIConroller I guess (like @tool4ever suggested)